### ISSUE TITLE
How to get backbone checkpoints of resnet18 (places365/cub)

### ISSUE THREAD

**Issue by ChoiDae1** [Trustworthy-ML-Lab/Label-free-CBM#6]
Hello!

I am looking for backbone checkpoints of ResNet18 that have been pre-trained on the Places365 and CUB datasets to reproduce your results. 
However, I have encountered difficulties in finding a clear source or repository that provides these specific backbone checkpoints. 

Thank you in advance for your support and guidance!

**Comment by tuomaso** [comment]
Hi, thanks for the question. For the Places365, we used the model from Network Dissection, [NetDissect-Lite](https://github.com/CSAILVision/NetDissect-Lite/tree/master) `script/dlzoo_example.sh`. However, this download link seems to no longer be working, so I have provided an alternative download link in `download_rn18_places.sh`. The CUB backbone should be automatically downloaded by the pytorchcv package (see `data_utils.py`).
